djstripe/stripe_objects.py

```diff
@@ -178,28 +181,47 @@ def _attach_objects_hook(self, cls, data):
         pass
 
     @classmethod
-    def _create_from_stripe_object(cls, data):
+    def _create_from_stripe_object(cls, data, save=True):
         """
-        Create a model instance using the given data object from Stripe
+        Instantiates a model instance using the provided data object received
+        from Stripe, and saves it to the database if specified.
+        :param data: The data dictionary received from the Stripe API.
         :type data: dict
+        :param save: If True, the object is saved after instantiation.
+        :type save: bool
+        :returns: The instantiated object.
         """
         instance = cls(**cls._stripe_object_to_record(data))
         instance._attach_objects_hook(cls, data)
-        instance.save()
-
+        if save:
+            instance.save()
         return instance
 
     @classmethod
-    def _get_or_create_from_stripe_object(cls, data, field_name="id"):
+    def _get_or_create_from_stripe_object(cls, data, field_name="id",
```

Comment:
That pushes it beyond 80 characters - Is that acceptable?  We usually keep ours below it in line with PEP-8 (and it is likely a common theme throughout any submits I've made), but this is your project and I'll follow whatever guidelines you lay out.